extensions/amp-video-docking/1.0/placeholder.js

```diff
@@ -0,0 +1,47 @@
+import * as Preact from '../../../src/preact';
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {fillStretch} from '../../amp-video/1.0/video-wrapper.css';
+import {useMemo} from '../../../src/preact';
+import {useStyles} from './dock.jss';
+
+export const Placeholder = ({styles}) => {
```

Comment:
Like the Controls component, this is supposed to look different based on element breakpoints (not done yet.)